lib/driver.js

```diff
@@ -622,6 +622,12 @@ class XCUITestDriver extends BaseDriver {
         await this.opts.device.setReduceTransparency(this.opts.reduceTransparency);
       }
 
+      if (_.isBoolean(this.opts.autoFillPasswords)) {
```

Comment:
I think this whole settings check block is too long and should be extracted to a separated method to improve the readability